src/app/components/shared/header/header.component.html

```diff
@@ -1,25 +1,28 @@
-<nav class="container-fluid navbar navbar-expand-lg navbar-dark fixed-top">
+<!--
+  Note: Using [routerLink] instead of [strongRoute] so that routerLinkActive will function.
+  Otherwise uses will not have a visual clue for which page they are on 
```

Comment:
sounds like there's a deficiency in strongRoute? 

File an issue?

Reply:
Tracking issue here: https://github.com/QutEcoacoustics/workbench-client/issues/1216